asserts/account_key.go

```diff
@@ -21,9 +21,12 @@ package asserts
 
 import (
 	"fmt"
+	"regexp"
 	"time"
 )
 
+var ValidAccountKeyName = regexp.MustCompile(`^(?:[a-z0-9]+-?)*[a-z](?:-?[a-z0-9])*$`)
```

Comment:
This should be moved into a function, `IsValidAccountKeyName` or similar at some point. Not a blocker here, though.